apis/Google.Cloud.PubSub.V1/docs/index.md

```diff
@@ -2,6 +2,15 @@
 
 {{description}}
 
+# Class renaming in [`v1.0.0-beta16`](https://www.nuget.org/packages/Google.Cloud.PubSub.V1/1.0.0-beta16)
```

Comment:
Have you checked what the code font in a link looks like? I seem to remember it's unpleasant and that we try to avoid it.

Reply:
Hmm, OK, I've removed code font in all links.